test/test-clusters/src/main/java/org/elasticsearch/test/cluster/local/FipsEnabledClusterConfigProvider.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.test.cluster.local;
+
+import org.elasticsearch.test.cluster.util.resource.Resource;
+
+public class FipsEnabledClusterConfigProvider implements LocalClusterConfigProvider {
+
+    @Override
+    public void apply(LocalClusterSpecBuilder builder) {
+        if (isFipsEnabled()) {
+            builder.configFile(
+                "fips_java.security",
+                Resource.fromClasspath(isOracleJvm() ? "fips/fips_java_oracle.security" : "fips/fips_java.security")
+            )
+                .configFile("fips_java.policy", Resource.fromClasspath("fips/fips_java.policy"))
+                .configFile("cacerts.bcfks", Resource.fromClasspath("fips/cacerts.bcfks"))
+                .systemProperty("java.security.properties", "${ES_PATH_CONF}/fips_java.security")
+                .systemProperty("java.security.policy", "${ES_PATH_CONF}/fips_java.policy")
```

Comment:
For the same reason as above, we should prefix with `=` char:
```suggestion
                .systemProperty("java.security.policy", "=${ES_PATH_CONF}/fips_java.policy")
```